### ISSUE TITLE
Correct the ReadMe file's Heading from "Applicarion " to Application

### ISSUE THREAD

**Issue by avinashji1302** [avinashji1302/Weather_Application_Using_Realtime_API#5]

**Comment by pulkitxm** [comment]
Can I get assigned? let me know how to start.

**Comment by avinashji1302** [comment]
> Can I get assigned? let me know how to start.

Yes you can work....And this issue has been solved but i am sure there is lots of typo in Readme file ...You can work

**Comment by Aayush-Kapur** [comment]
Hey, I think there's some mismatch in your project's ReadMe file, 
first : small typos
second : title and description don't really go together , title and codebase says Weather Application , readme says "Todo App" 

If you think is is one of the issues, I would love to contribute to this
